src/stylesheets/core/mixins/_icon.scss

```diff
@@ -124,8 +124,8 @@
   );
 
   $mask-props: url("#{$path}/usa-icons/#{$filename-base}.svg") no-repeat center /
-    #{$width} #{$height};
-  $image-props: url("#{$path}/#{$filename-ie11}") no-repeat center / #{$width} #{$height};
+    contain;
+  $image-props: url("#{$path}/#{$filename-ie11}") no-repeat center / contain;
```

Comment:
Are the `$width` and `$height` variables unused now after these changes? (i.e. can they be removed?)

Oddly, there's also an assignment a few lines below this line, which I'm not sure would have had any effect? 🤔

Reply:
`$width` and `$height` are used to get consistent sizing of the icons across all browsers in both background and mask.

It seemed more straightforward and allowed consistent rendering this way. But open to any improvements.

Reply:
Ohhh, my brain misread line 134 as `$width:` (variable assignment) instead of `width:` (CSS property). Disregard 🤦

Reply:
All good! Just getting back from break so the brain fog on this was real heavy. 😅